README.md

```diff
@@ -1,7 +1,7 @@
-# snap collector plugin - load
+# Snap collector plugin - load
 This plugin collects metrics from /proc/loadavg kernel interface about load average figures giving the number of jobs in the run queue (state R) or waiting for disk I/O (state D) averaged over 1, 5, and 15 minutes.  
 
-It's used in the [snap framework](http://github.com:intelsdi-x/snap).
+It's used in the [Snap framework](http://github.com:intelsdi-x/snap).
 
 1. [Getting Started](#getting-started)
   * [System Requirements](#system-requirements)
```

Comment:
Please correct below the following line:
```
 * [Configuration and Usage](configuration-and-usage)
```
You need to add `#` before configuration-and-usage.